www/src/pages/ar/folder-structure.mdx

```diff
@@ -6,227 +6,267 @@ lang: ar
 
 <sub>(في حالة NextAuth.js + tRPC)</sub>
 
-#### ملف `src/server/db/client.ts`
+</div>
+<div data-components="prisma">
+
+#### `src/server/db/client.ts`
 
 ملف `client.ts` يستخدم لإنشاء نُسخة لـ Prisma Client علي مستوي التطبيق كلة، إقرأ المزيد [Prisma usage](usage/prisma#prisma-client)
 
 <sub>(في حالة Prisma)</sub>
 
-### ملف `src/server/trpc`
+</div>
+<div data-components="trpc">
+
+### `src/server/trpc`
 
 مجلد tRPC يحتوي على كود tRPC للسيرفر
 
 <sub>(في حالة tRPC)</sub>
 
-#### ملف `src/server/trpc/context.ts`
+</div>
+<div data-components="trpc">
+
+#### `src/server/trpc/context.ts`
 
 ملف `context.ts` يُستخدم لإنشاء context في كل طلب tRPC، إقرأ [tRPC usage](usage/trpc#-servertrpccontextts)
 
 <sub>(في حالة tRPC)</sub>
 
-#### ملف `src/server/trpc/trpc.ts`
+</div>
+<div data-components="trpc">
+
+#### `src/server/trpc/trpc.ts`
 
 إقرأ [tRPC usage](usage/trpc#-servertrpctrpcts)
 
 <sub>(في حالة tRPC)</sub>
 
-### ملف `src/server/trpc/router`
+</div>
+<div data-components="trpc">
+
+### `src/server/trpc/router`
 
 يحتوي على `tRPC routers`
 
 <sub>(في حالة tRPC)</sub>
 
-#### ملف `src/server/trpc/router/_app.ts`
+</div>
+<div data-components="trpc">
+
+#### `src/server/trpc/router/_app.ts`
 
 ملف `_app.ts` يُستخدم لدمج tRPC routers في router واحد كما هو الحل في الـ types، إقرأ المزيد هنا [tRPC usage](usage/trpc#-servertrpcrouterts)
 
 <sub>(في حالة tRPC)</sub>
 
-#### ملف `src/server/trpc/router/auth.ts`
+</div>
+<div data-components="nextauth+trpc">
+
+#### `src/server/trpc/router/auth.ts`
 
 ملف `auth.ts` هو مثال لاستخدام `protectedProcedure helper` في حماية route معين.
 <sub>(في حالة NextAuth.js + tRPC)</sub>
 
-#### ملف `src/server/trpc/router/example.ts`
+</div>
+<div data-components="trpc">
+
+#### `src/server/trpc/router/example.ts`
 
 ملف `example.ts` هو مثال لإستخدام `publicProcedure` لإنشاء public tRPC route
 
 <sub>(في حالة tRPC)</sub>
 
-### ملف `src/styles`
+</div>
+<div>
+
+### `src/styles`
 
 مجلد `styles` يحتوي على الـ stylesheets
 
 <sub>(في حالة Tailwind CSS)</sub>
 
-### ملف `src/types`
+</div>
+<div data-components="nextauth">
+
+### `src/types`
 
 مجلد `types` يحتوي على الـ Types
 
 <sub>(في حالة NextAuth.js)</sub>
 
-#### ملف `src/types/next-auth.d.ts`
+</div>
+<div data-components="nextauth">
+
+#### `src/types/next-auth.d.ts`
 
 ملف `next-auth.d.ts` يُسنخدم للتعديل على إعدادات NextAuth الافتراضية، لمزيد من المعلومات [NextAuth.js usage](usage/next-auth#inclusion-of-userid-on-the-session).
 
 <sub>(في حالة NextAuth.js)</sub>
 
-### ملف `src/utils`
+</div>
+<div data-components="trpc">
+
+### `src/utils`
 
 مجلد `utils` يُستخدم للـ functions التي يكثر إستخدامها.
 
 <sub>(في حالة tRPC)</sub>
 
-#### ملف `src/utils/trpc.ts`
+</div>
+<div data-components="trpc">
+
+#### `src/utils/trpc.ts`
 
 ملف `trpc.ts` يُسخدم كـ entrypoint لـ tRPC، إقرأ المزيد هنا [tRPC usage](usage/trpc#-utilstrpcts)
 
 <sub>(في حالة tRPC)</sub>
 
-### ملف `.env`
+</div>
+<div>
+
+### `.env`
```

Comment:
Random question, should this be heading two? `##`

I think the heading is used here to indicate where the file is, so if it's heading 2, then it's in the root section of the app.